tests/test_parse_anything.py

```diff
@@ -30,3 +30,4 @@ def test_parse_anything(test_file):
     assert doc.name in ('Sample_Document-V2.1', 'xyz-0.1.0', 'SPDX-Tools-v2.0')
     assert doc.comment in (None, 'This is a sample spreadsheet', 'Sample Comment',
                            'This document was created using SPDX 2.0 using licenses from the web site.')
+    assert len(doc.packages) in (4, 4, 1, 2, 4, 3)
```

Comment:
I am not sure if this assert statement really makes sense. You could simply write `doc.packages in [1,2,3,4]`. But this would lead to a passing test if for example one of 4 packages is not parsed because 3 is still in the list and we don't have a mapping between the files and the asserted count of packages. The whole test coverage is not that best but I think this statement wouldn' really improve it, so for now asserting that there is no error while parsing is sufficient.